### ISSUE TITLE
query all commitments and intents returns duplicate entries

### ISSUE THREAD

**Issue by weswalla** [holo-rea/holo-rea#328]
This error is not produced consistently.
When running the tests in the following file,
https://github.com/h-REA/hREA/blob/bade910615e6f8fae43267457ec4d430ccee4f3d/test/process/process_records_e2e.js#L153-L169
some runs pass, where others return 10 edges instead of 2, but its just duplicates of the 2. For example, the console log is the following:
```
all commitments [
  [Object: null prototype] {
    node: [Object: null prototype] {
      id: 'uhCEk8H9-0be3oS0Lo_PRnb06TNQHas9yOxxBrLqeeF-92P-_KduF:uhC0kq7vuYPUHEXEqs8qmFfy3iy52scov-jO1UUyTfHLMmm7lG7Ma'
    }
  },
  [Object: null prototype] {
    node: [Object: null prototype] {
      id: 'uhCEkebWRO24hw4B7l984cztnc5UTK_5G7bCONXEykasUJRSuTE-9:uhC0kq7vuYPUHEXEqs8qmFfy3iy52scov-jO1UUyTfHLMmm7lG7Ma'
    }
  },
  [Object: null prototype] {
    node: [Object: null prototype] {
      id: 'uhCEk8H9-0be3oS0Lo_PRnb06TNQHas9yOxxBrLqeeF-92P-_KduF:uhC0kq7vuYPUHEXEqs8qmFfy3iy52scov-jO1UUyTfHLMmm7lG7Ma'
    }
  },
  [Object: null prototype] {
    node: [Object: null prototype] {
      id: 'uhCEkebWRO24hw4B7l984cztnc5UTK_5G7bCONXEykasUJRSuTE-9:uhC0kq7vuYPUHEXEqs8qmFfy3iy52scov-jO1UUyTfHLMmm7lG7Ma'
    }
  },
  [Object: null prototype] {
    node: [Object: null prototype] {
      id: 'uhCEk8H9-0be3oS0Lo_PRnb06TNQHas9yOxxBrLqeeF-92P-_KduF:uhC0kq7vuYPUHEXEqs8qmFfy3iy52scov-jO1UUyTfHLMmm7lG7Ma'
    }
  },
  [Object: null prototype] {
    node: [Object: null prototype] {
      id: 'uhCEkebWRO24hw4B7l984cztnc5UTK_5G7bCONXEykasUJRSuTE-9:uhC0kq7vuYPUHEXEqs8qmFfy3iy52scov-jO1UUyTfHLMmm7lG7Ma'
    }
  },
  [Object: null prototype] {
    node: [Object: null prototype] {
      id: 'uhCEk8H9-0be3oS0Lo_PRnb06TNQHas9yOxxBrLqeeF-92P-_KduF:uhC0kq7vuYPUHEXEqs8qmFfy3iy52scov-jO1UUyTfHLMmm7lG7Ma'
    }
  },
  [Object: null prototype] {
    node: [Object: null prototype] {
      id: 'uhCEkebWRO24hw4B7l984cztnc5UTK_5G7bCONXEykasUJRSuTE-9:uhC0kq7vuYPUHEXEqs8qmFfy3iy52scov-jO1UUyTfHLMmm7lG7Ma'
    }
  },
  [Object: null prototype] {
    node: [Object: null prototype] {
      id: 'uhCEk8H9-0be3oS0Lo_PRnb06TNQHas9yOxxBrLqeeF-92P-_KduF:uhC0kq7vuYPUHEXEqs8qmFfy3iy52scov-jO1UUyTfHLMmm7lG7Ma'
    }
  },
  [Object: null prototype] {
    node: [Object: null prototype] {
      id: 'uhCEkebWRO24hw4B7l984cztnc5UTK_5G7bCONXEykasUJRSuTE-9:uhC0kq7vuYPUHEXEqs8qmFfy3iy52scov-jO1UUyTfHLMmm7lG7Ma'
    }
  }
]
```

**Comment by weswalla** [comment]
@pospi any idea why this is happening?

**Comment by pospi** [comment]
No. @Connoropolous is there anything in the Tryorama setup that might be reusing conductor working directories?

I did encounter a weird situation a couple of days ago where it seemed as though I was getting old WASM errors being returned from a previous test run... without changing any code I re-ran and they went away for no discernable reason.

**Comment by Connoropolous** [comment]
Not that I know of. Let me run a few times see if i can reproduce

**Comment by Connoropolous** [comment]
I have a theory about this, now that I've just seen something similar. 
I believe it may be occurring when multiple people are executing code against the tests at or around the same time. This would mean gossip, and that would mean extra entries. bizarre but true. It's actually happened to me before in other circumstances. I just tweaked my local `uid` property of dna bundles, repackaged, and my error went away.

**Comment by pospi** [comment]
lol that's hilarious. So even sandboxed conductors gossip across the globe? Seems weird. I'm gonna run this by Eric (:

**Comment by Connoropolous** [comment]
yes it looks like the default network type is quic
https://github.com/holochain/tryorama/blob/31f5fd86cc98f1e290ef7deed176b13aa79d6ea2/ts/src/local/conductor.ts#L188

it might not gossip across the internet without a proxy, but it's possible it could make a network connection with the other CI test runners which are running in parallel.

**Comment by Connoropolous** [comment]
the `sandbox` doesn't necessarily imply non network active
